kong/plugins/http-log/sender.lua

```diff
@@ -7,6 +7,7 @@ local sender = {}
 
 local ngx_encode_base64 = ngx.encode_base64
 local ERR = ngx.ERR
+print('1')
```

Comment:
Will you please remove this undesired change? Thanks!

Reply:
OK